mainnetSubnetList.json

```diff
@@ -0,0 +1,77 @@
+[
```

Comment:
I wonder if it is worth changing all of our list top level items from an array to an object indexed by the canonical id for the list (e.g. subnetId / chainId)?

Without this, the consumers of this will likely need to build a map themselves which someone ruins the purpose of denormalizing all the data here in the first place.

Reply:
Yes I've considered this too.  I actually have a helper function in the subnet explorer that reindexes these arrays as objects exactly as you describe.  Only problem is that sometimes we need arrays indexed by different values, depending on how the frontend wants to display/group the data (though usually it _is_ indexed by chainId or subnetId).  The current format is more agnostic which is kinda nice, and also it allows for ordered responses (which isn't something we do yet, but sorting the lists is something on the table in the future).

All said, I'm definitely open to the idea of turning these into object-formats so we can get juicy constant-time lookups.  Definitely worth the performance gain in many places.  Alternatively, the API could handle embedding the data from this repo into individual responses, so that referencing these JSONs directly were unnecessary.  Just a thought 😉.

In any case, let's consider this change in a separate PR since it would cause a bit too much scope creep.

Reply:
We're somewhat already on it with regard to embedding this in the api responses.  Again would be even better if we had a graphql layer to stitch these things in when needed by the caller.  I think we both agree that is the direction we should probably go next with the api.

Sounds good leaving this out.  I'm aligned with your thoughts here and see why indexing might cause other issues.